### ISSUE TITLE
Sponsors Directory for external sponsorship opportunities for events

### ISSUE THREAD

**Issue by elioqoshi** [mozilla/participation-org#254]
Goal: Create and facilitate a tool documenting sponsorship opportunities from external sponsors for Mozilla related events, to help Mozillians when reaching out for event support.

Success: Have a directory of at least 50 sponsors by end of Q1 accessible to Mozillians.

[What would success look like]
Related Issues & Links:

FOSDEM
OSCON
OSCAL

other events (check sponsors)

**Comment by francjp** [comment]
Hi @elioqoshi,

I have a couple of questions related to this:
- Why these 3 events as example? (I would add JS conferences, for example)
- This is for traveling or for Sponsor the event?
- FOSDEM is in January 30th, I would suggest to include it for 2017 edition in any case.
- OSCON will be more complicated, since it will happen in Europe too, and we would like to have a consistent plan for participate there.
- Is there any idea about how the events will be selected, and how will be the planning for Mozillians supported?
- Why do you think that mozillians should look "outside" to get support?
- Once you have the list, how the mozillians will contact these external sponsors? Where it will be published, and how will be the procedure?

I think that this is a good idea, but need a broader plan (I would think this is as a starting point). Also, I think that the plan should include some kind of debrief or follow-up of these people, since we want them to have impact.

Cheers,

**Comment by shadowcrow** [comment]
I would like recommend COSCUP[1], or g0v summit. :D 
I have interested to sponsor Mozillian to other conference, it makes people have different connection.
As COSCUP 2015 organizer, we tried to invite some Mozillians from East south Asia, but unfortunately he couldn't get travel sponsor from Mozilla. I wish we can discuss more about this kind of situation. :D

[1]http://coscup.org/2015/en/
[2]http://summit.g0v.tw/

**Comment by elioqoshi** [comment]
Hey @francjp 

I think we have a miscommunication here, I did this quickly at the last Participation session at the work week, so excuse me.

The idea is to have a list of external sponsors (not Mozilla) who can sponsor Mozilla events organized by the community. So if I want to organize a Firefox OS hackathon, I'd know whom to approach to sponsor the event (even partially) so Mozilla's costs can be reduced.

> Why these 3 events as example? (I would add JS conferences, for example)

Just personal biased examples I have experience with. Only for illustration purposes.

> This is for traveling or for Sponsor the event?

I suppose event organizing. It would make sense that other sponsors would be interested to sponsor the food for example, even if it's a Mozilla event.

> Is there any idea about how the events will be selected, and how will be the planning for Mozillians supported?

This could be something Mozillians (especially Reps) themselves can reach out to, not related to official processes we already have.

---

Regarding the other points, as I said, it was mostly for events organized by that said Mozillian. We for example found this really useful for OSCAL where Mozilla had the biggest presence and focus (so yeah, I'm not talking regarding super specific Mozilla events, but events where Mozilla is a focus, yet other sponsors might want to support too).

> Once you have the list, how the mozillians will contact these external sponsors? Where it will be published, and how will be the procedure?

Not sure. Do you think we need to track this on a case by case basis?

**Comment by francjp** [comment]
Hi @elioqoshi,

> The idea is to have a list of external sponsors (not Mozilla) who can sponsor Mozilla events organized by the community. So if I want to organize a Firefox OS hackathon, I'd know whom to approach to sponsor the event (even partially) so Mozilla's costs can be reduced.

Ok, then the examples are confusing, since FOSDEM is not sponsored by Mozilla (we don't pay anything to the organization). While OSCON is more complicated, since they usually ask above 10 K only for sponsoring.

In that case, and based on the examples included here (+COSCUP) I think that you should create 2 issues:
- One for events organized by Mozillians that need sponsoring
- Other for events that Mozillians will attend.

You already have the experience from OSCAL, but I don't know how many mozillians are organizing events like that, and needs sponsoring. Aside that, based on what you will request sponsor for one event, and how the amount will be decided?, sorry, I'm trying to imagine more than one event, and each one will have different approach, and different "interesting" sponsors to reach.

In any case, I do think that this is somehow Reps related and since you already know, we are trying not to focusing only in events. Perhaps we should discuss this in another channel?, It sounds to me like one of the reasons is that we are not sponsoring Reps, or events organized by Reps, so I'm concern that it looks like that (from both sides). If this is not the case you are rising, accept my apologies.

I would rather prefer focusing on the first point and try to have a list of events first, and then the list of "partners"

For the second point it will be more tricky, since there are more factors involved:
- Selection process (you will need a template for this, or a procedure documented)
- Amount of people (this will have to be for each event, and if it includes booth or not)
- Assets needed (if it's a technical event, you wouldn't send people from marketing, etc)
- If it's open to everyone, or to some communities (long distances makes trips more expensive)

And this without thinking of it too much. At least these are my starting points each time we help to coordinate an event.

> Regarding the other points, as I said, it was mostly for events organized by that said Mozillian. We for example found this really useful for OSCAL where Mozilla had the biggest presence and focus (so yeah, I'm not talking regarding super specific Mozilla events, but events where Mozilla is a focus, yet other sponsors might want to support too)

I would like to see other examples aside OSCAL, since I think that it can be confusing only putting this as an example.

> Not sure. Do you think we need to track this on a case by case basis?

Well, personally I don't want to see Mozilla wasting money, and I don't want to see other companies doing that too. And as far as I know, everyone will want to see impact, goals, metrics. If not, and apologies for be so direct, it's traveling only because someone ask money for that.

So yes, this will have to be tracked case by case, and have a proper follow-up (like Reps usually ask). So from my point of view, the issue is interesting, but it's not so simple (and that's why it looks interesting).

Regards,

**Comment by elioqoshi** [comment]
Hey @francjp 

Sorry for being unclear again. I mentioned these events as good sources we can list sponsors on. FOSDEM and OSCON has a great list of sponsors which might also be interested to sponsor other independent events by Mozillians. It's not related whether Mozilla sponsors them or not. I hope I got my point across now.

I again think we missunderstood each other. To recap again, my idea was to have a list of potential companies who are interested to sponsor events organized by Mozillians. Mozillians can reach out to them individually then. I only mentioned these conferences as they have an extensive list of sponsors which can be worth reaching out when looking for event sponsors.

This is not to sponsor Mozillians get to attend events (at least that was not my idea when I wrote the issue), but to partially fund Mozillians organized events to be successful.

This is mostly Reps specific, but I consulted with @rosanardila if it would make sense to open an issue here and she was positive about it.

Again, I hope I got the idea across as I think there was some confusion.
